.github/workflows/build.yml

```diff
@@ -0,0 +1,12 @@
+name: build
+on: [push]
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
Doesn't matter in this repo, but prob better to use v2 (it doesn't checkout entire repo, but just HEAD commit by default)

Reply:
copy pasted this from one of my side projects, thanks